### DISCUSSION TITLE
Henry Zhu -- Babel, Open Source

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#116] [Announcements]
## Background

- [twitter](https://twitter.com/left_pad)
- [Website](https://www.henryzoo.com/)
- [GitHub](https://github.com/hzoo)
- [hopeinsource podcast](https://hopeinsource.com/)

## Themes

- Open source sustainability
- babel

## Intro

**Andrew**: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

**Justin**: Hey everyone! Our guest today is Henry Zhu. You might know Henry as a long time maintainer and steward of the babel project. Henry, welcome, it's so great to have you on the show. Before we start, would you like to tell our audience a bit more about yourself?

### General OSS

- [x] How did you get involved with open source?
- [x] What was it like running babel such a large open source project?
- [x] What's your role in babel these days?
- [x] When did you decide to make the full transition to live solely off sponsorships?
- [x] Have you dealt with burn out?
- [x] What have you been up to lately in OSS?

### Babel

- [ ] ~Are there any plans for babel v8?~
- [ ] ~Do you think that you will need to maintain babel forever? Will evergreen browser get rid of much of the need? How do you feel about new projects like swc that aim to replace it?~
- [x] Thanks for you involvement in AST explorer. its a tool we really love! It's probably helped thousands of people create babel plugins. What's the most creative thing you've done with ASTs?

### Future

- [x] What do you think ideal scenario is for open source funding at scale?
- [x] What do you think the future of open source is? (@zephraph any suggestions?)

**Comment by hipstersmoothie** [comment]
https://github.com/gbj/leptos
https://github.com/CamHenlin/coprocessor.js + https://github.com/CamHenlin/MessagesForMacintosh

**Comment by hzoo** [comment]
https://github.com/lucax88x/CodeAceJumper

https://github.com/openai/whisper, https://github.com/ggerganov/whisper.cpp, https://github.com/tobiashuttinger/openai-whisper-realtime

https://twitter.com/MatthewWSiu/status/1560725842019368962

https://twitter.com/left_pad/status/1371255314151051264
https://twitter.com/left_pad/status/1531805954899169284

**Comment by just-be-dev** [comment]
Tooltips:
- https://alexanderobenauer.com/labnotes/000/
- https://phiresky.github.io/blog/2021/hosting-sqlite-databases-on-github-pages/
